internal/integration-tests/equipment/integration_test.go

```diff
@@ -501,10 +503,16 @@ func TestIntegration_BlockEquipment(t *testing.T) {
 	if testing.Short() {
 		t.Skip("skipping integration test")
 	}
+	loc := time.UTC
+	tm := time.Now().In(loc)
+
+	// In these number of days the blocking will start/end
+	const startNumDays = 1
+	const endNumDays = 10
```

Comment:
No need for const usage in testing func. Const should be putted in top segment of file and have a global usage. In functions it is much better to just use a variable
```suggestion
        startNumDays := 1
	endNumDays := 10
```